workflowtrigger/util/auditlog/log.go

```diff
@@ -113,23 +115,38 @@ func UnmarshallString(str string) Log {
 	return Unmarshall(m)
 }
 
-func Marshal(log Log) map[string]interface{} {
+func Marshal(log Log) *ordered.OrderedMap {
+	mapObj := ordered.NewOrderedMap()
 	switch log.(type) {
 	case *RemediationLog:
-		mapObj := serializerutil.GetMapFromStruct(log)
-		mapObj["type"] = serializerutil.RemediationLogType
-		return mapObj
+		remLog := log.(*RemediationLog)
+		mapObj.Set("pipelineUniqueVersionNumber", remLog.PipelineUniqueVersionNumber)
+		mapObj.Set("uniqueVersionInstance", remLog.UniqueVersionInstance)
+		mapObj.Set("unhealthyResources", remLog.UnhealthyResources)
+		mapObj.Set("remediationStatus", remLog.RemediationStatus)
+		mapObj.Set("type", serializerutil.RemediationLogType)
```

Comment:
For the remediation log and the deployment log, put the type as the first component.

Reply:
Moved `type` field on the first place in those objects.